### PR TITLE
Added mock continuum login service fixed continuum-auth token parsing

### LINKED ISSUE #212: Add mock Continnum login
Add a mock microservice to be used for local development to mock out logging into to continuum.

Conintuum has a container `api-dummy` for the api but nothing for logging in (need to confirm). This service will respond to an endpoint (TBD) with a valid token for continuum.

## Reference 
https://api.elifesciences.org/documentation/
`docker pull elifesciences/api-dummy`

### PR DESCRIPTION
**Pull request by erezmus**
This PR adds a basic mock continuum auth service. It essentially starts the login process by creating a journal token to be resigned by continuum-auth which now redirects to the client app with the reviewer token

Closes #212

### CODE REVIEW

File: mock-continuum-login/src/use-cases/submit.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    const token = sign({
+        iss: "journal--prod",
+        id: "ewwboc7m",
+        "new-session": true
+    },
+    'some_secret_from_journal', {
+        expiresIn: '1d',
+    })
+
+    resp.redirect(`http://localhost:3001/authenticate/${token}`);
```
Comment (diversemix) [inline mock-continuum-login/src/use-cases/submit.ts]: Be good to make this config

File: mock-continuum-login/src/use-cases/submit.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import { sign } from "jsonwebtoken";
+
+export const Submit = (req: Request, resp: Response) => {
+    const token = sign({
+        iss: "journal--prod",
+        id: "ewwboc7m",
+        "new-session": true
+    },
+    'some_secret_from_journal', {
+        expiresIn: '1d',
```
Comment (diversemix) [inline mock-continuum-login/src/use-cases/submit.ts]: I think the expiry from journal is longer than this? @petereast ?

File: mock-continuum-login/src/use-cases/submit.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { Request, Response } from "express";
+import { sign } from "jsonwebtoken";
+
+export const Submit = (req: Request, resp: Response) => {
```
Comment (diversemix) [inline mock-continuum-login/src/use-cases/submit.ts]: I think we should write a test for this file.

File: mock-continuum-login/src/use-cases/submit.ts (hunk 2 above)
Comment (erezmus) [reply mock-continuum-login/src/use-cases/submit.ts]: https://github.com/elifesciences/elife-xpub/blob/c383f75960c03febf387123d80409df3c63252bc/packages/component-elife-app/server/resolvers.js#L15

we have it at one day currently

**Pull request merged**